test/math/FloatSize2DTest.ooc

```diff
@@ -90,10 +90,62 @@ FloatSize2DTest: class extends Fixture {
 			expect(FloatSize2D basisX angle(-FloatSize2D basisY), is equal to(-PI as Float / 2.0f) within(this precision))
```

Comment:
`PI as Float` is used in several places, I will replace all of them with `Float pi`